packages/adapter-cloudflare/index.js

```diff
@@ -144,29 +144,34 @@ export default function (options = {}) {
 				);
 			}
 		},
-		async emulate() {
-			const proxy = await getPlatformProxy(options.platformProxy);
-			const platform = /** @type {App.Platform} */ ({
-				env: proxy.env,
-				context: proxy.ctx,
-				caches: proxy.caches,
-				cf: proxy.cf
-			});
+		emulate() {
+			// we want to invoke `getPlatformProxy` only once and await it only when it is accessed
```

Comment:
```suggestion
			// we want to invoke `getPlatformProxy` only once and await it only when it is accessed
			// If we would await it here, it would hang indefinetly because the platform proxy only resolves once a request happens
```